### PR TITLE
adding ended event data for AudioScheduledSourcenode and MediaStreamTrack

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://github.com/mdn/sprints/issues/1151

For the two event data additions, I just copied the data from their associated `on...` properties.

### CODE REVIEW

File: api/AudioScheduledSourceNode.json [hunk 1]
```diff
@@ -99,6 +99,79 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioScheduledSourceNode/ended_event",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": "14",
+                "version_removed": "56",
+                "notes": "Before version 57, this event was implemented on  AudioBufferSourceNode, OscillatorNode, and ConstantSourceNode, which are now children of this class."
```
Comment (jpmedley) [inline api/AudioScheduledSourceNode.json]: There is an extra space between 'on' and 'AudioBufferSourceNode'.

File: api/AudioScheduledSourceNode.json [hunk 2]
```diff
@@ -99,6 +99,79 @@
...
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": "18",
+                "version_removed": "56",
+                "notes": "Before version 57, this event was implemented on  AudioBufferSourceNode, OscillatorNode, and ConstantSourceNode, which are now children of this class."
```
Comment (jpmedley) [inline api/AudioScheduledSourceNode.json]: Here too.

File: api/AudioScheduledSourceNode.json [hunk 3]
```diff
@@ -99,6 +99,79 @@
...
+              "version_added": true
+            },
+            "webview_android": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": true,
+                "version_removed": "56",
+                "notes": "Before version 57, this event was implemented on  AudioBufferSourceNode, OscillatorNode, and ConstantSourceNode, which are now children of this class."
```
Comment (jpmedley) [inline api/AudioScheduledSourceNode.json]: And here.

**Comment by chrisdavidmills** [comment]
Niggly spacing issues now fixed. Cheers @jpmedley !

**Comment by chrisdavidmills** [comment]
Thanks folks; I guess we can merge this then. Does one of you want to do the honour? I'd say it is better if I don't merge my own PRs ;-)

@vinyldarkscratch yeah, this does look wrong. Maybe file an issue for this?

**Comment by vinyldarkscratch** [comment]
Rather than an issue, I did one better and submitted a PR, particularly #3894.  :wink:

As for the merge, well, might as well hit that nice, big, tempting button, right?

**Comment by chrisdavidmills** [comment]
@vinyldarkscratch awesome, nice one!

And yes, it is approved, so please do.

**Pull request merged**